Source/JavaScriptCore/assembler/X86Assembler.h

```diff
@@ -2393,6 +2398,49 @@ class X86Assembler {
         m_formatter.immediate8((uint8_t) laneIndex);
     }
 
+    void pshufd(XMMRegisterID vd, XMMRegisterID vn, uint8_t controlBits)
```

Comment:
Convention-wise, is this matching the manual? I thought intel usually put dest on the right. David, Yusuke?

Reply:
Putting dest is the macro assembler's convention. In each arch's assembler, we rather follow each arch's assembly lang's convention. So align it to the other existing Intel assembler's method. Since Intel assembly
 lang uses dest on the left, we follow here.

Reply:
Summary: follow to the X86Assembler class's convention.